pkg/controller/dataimportcron-controller.go

```diff
@@ -241,6 +241,7 @@ func getCronRegistrySource(cron *cdiv1.DataImportCron) (*cdiv1.DataVolumeSourceR
 	return source.Registry, nil
 }
 
+// FIXME: refactor and adapt for concurrent imports
```

Comment:
What does it mean? Is it a short term goal, to do before PR is merged or is it a long term and already in plans with an active issue? 

In my experience such fixme is lost without an issue.

Reply:
Short-medium term goal, not for this PR.

Reply:
So maybe create issue here or plan it somewhere. :) I am sure you can find at least one of my old FIXME or TODO which was forgotten.

Maybe I should scan the code for this ;)